mesonbuild/modules/windows.py

```diff
@@ -183,7 +183,7 @@ def get_names() -> T.Iterable[T.Tuple[str, str, T.Union[str, mesonlib.File, buil
             # Path separators are not allowed in target names
             name = name.replace('/', '_').replace('\\', '_').replace(':', '_')
             name_formatted = name_formatted.replace('/', '_').replace('\\', '_').replace(':', '_')
-            output = f'{name}_@BASENAME@.{suffix}'
+            output = f'@BASENAME@.{suffix}'
```

Comment:
If you drop `name` here, it becomes an unused variable, so logically we should also drop it from get_names() as well.